config/networking/networkpolicy-operator.yaml

```diff
@@ -20,4 +20,4 @@ spec:
         - protocol: TCP
           port: 8081
         - protocol: TCP
-          port: 9443
\ No newline at end of file
+          port: 9443
```

Comment:
the podSelector matches for `app.kubernetes.io/name: fluent-bit`. This looks wrong. Can anybody confirm this?